Assets/_BForBoss/_Weapons/Scripts/RifleBulletBehaviour_NoPhysics.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Perigon.Weapons
+{
+    public class RifleBulletBehaviour_NoPhysics : BulletBehaviour
```

Comment:
No need to call the specifically Rifle. most bullets behavior are the same
But name is `NoPhysicsBulletBehaviour`